src/utils/AccurateMassSearch.cpp

```diff
@@ -143,8 +142,8 @@ class TOPPAccurateMassSearch :
     // copy top-level params to algorithm
     ams_param.setValue("db:mapping", ListUtils::create<std::string>(getStringList_("db:mapping")));
     ams_param.setValue("db:struct", ListUtils::create<std::string>(getStringList_("db:struct")));
-    ams_param.setValue("positive_adducts", getStringOption_("positive_adducts"));
-    ams_param.setValue("negative_adducts", getStringOption_("negative_adducts"));
+    ams_param.setValue("positive_adducts", ListUtils::create<std::string>(getStringList_("positive_adducts")));
```

Comment:
pheww...
this conversion is not really elegant.
Why is there no overload?

Reply:
hahaha had a long discussion with @timosachsenberg 
I'll see what I can do